rfcs/0001-notary-integration.md

```diff
@@ -0,0 +1,39 @@
+# Problem
+kpack does not handle image signing.
+
+# Outcome
+kpack integrates with notary to sign images that it builds.
+
+# Actions to take
+There are a few concrete steps to make this work in kpack:
+1. Add a new section to the image config that allows users to configure notary settings:
+  ```
+  notary:
```

Comment:
This is `spec.notary` I assume? What would be the pros/cons of nesting this under v1 to better prepare us for future versions of notary?